infrastructure_overview/stream_data_integration.md

```diff
@@ -0,0 +1,11 @@
+# SemWES Stream Data Integration
```

Comment:
From this short text and the readme(s) in the code example, I would not be able to set this up.
Especially the connection to SemWES remains unclear to me. Maybe an illustration of how a workflow would look like will help, or a real-life example. Also, it's not clear to me where the SemWES services are hosted/deployed.